src/api/server/mod.rs

```diff
@@ -35,7 +35,10 @@ mod async_io;
 mod sync_io;
 
 /// Maximum buffer size of FUSE requests.
+#[cfg(target_os = "linux")]
 const MAX_BUFFER_SIZE: u32 = 1 << 20;
+#[cfg(target_os = "macos")]
+const MAX_BUFFER_SIZE: u32 = 1 << 25;
 const DIRENT_PADDING: [u8; 8] = [0; 8];
```

Comment:
Please move this into linux_abi.rs too:)

Reply:
`MAX_BUFFER_SIZE` is not a kernel ABI. Instead, it is part of the server API that can be negotiated with the kernel to set the max IO buffer.